src/js/popup.js

```diff
@@ -635,21 +636,25 @@ $(document).ready(function () {
         }
     });
 
-    // When the edit template has been compiled
+    // When the sandbox compiles a template
     $(window).on('message', function (event) {
         event = event.originalEvent;
-        if (event.data.html) {
-            $('#templateEditor').append(event.data.html);
+        if (event.data.content.localeCompare('template-editor') === 0) {
```

Comment:
Can this just use standard string comparison?
```
event.data.content === 'template-editor'
```

Reply:
👍